pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "modis-tools"
+version = "0.1.0"
+description = "An enivornment for modis tools to download MODIS imagery from the NASA Earthdata platform"
+authors = ["Chris <chris.c.earle@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.7.17" 
+requests = "2.*"
+pydantic = "1.*"
+python-dateutil = "2.*"
+shapely = "2.*"
+tqdm = "^4.42.0"
+setuptools = {version = "69.2.0", markers = "python_version >= '3.12'"}
```

Comment:
This file is missing two extras:

    extras_require={
        "test": ["pytest"],
        "gdal": ["gdal"],
    }